app/views/TaxCodeDetailsView.scala.html

```diff
@@ -21,102 +21,98 @@
 @import includes.ptaHeader
 @import includes.link
 
-@this(mainTemplate: MainTemplate)
+@this(main: MainTempleteNew)
 
 @(taxCodeViewModel: TaxCodeViewModel)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
 
 @header = @{
     ptaHeader(
-    displayBackLink = true,
+    displayBackLink = false,
     mainHeadingText = taxCodeViewModel.title,
-    preHeadingText = taxCodeViewModel.preHeader)
+    preHeadingText = taxCodeViewModel.preHeader,
+    showGdsVersion = true)
 }
 
-@mainTemplate(
+@main(
     title = taxCodeViewModel.title,
-    authedUser = Some(user),
-    pageHeader = Some(header),
-    articleClasses = Some("full-width")
-    ){
-
-    <div class="grid-row">
-        <div class="grid grid-2-3">
-            <div class="inner-block">
-
-                <div class="section flush--bottom">
-                    <p id="ledeMessage">
-                        <span class="lede">@taxCodeViewModel.ledeMessage</span>
-                    </p>
-                </div>
+    pageTitle = Some(Messages("tai.service.navTitle")),
+    authedUser = Some(user)
```

Comment:
did we agree on passing in backLinkUrl as the exact URL so as not to use javascript:history.go(-1)

Reply:
I asked Tarik and he told me to use JS